src/main/java/io/sentry/samples/example/Application.java

```diff
@@ -32,53 +33,43 @@
 	private static Map<String, Integer> inventory = new HashMap<>();
 
 	private void checkout(List<Item> cart) {
+
 		Map<String, Integer> tempInventory = inventory;
 		for(Item item : cart) {
 			int currentInventory = tempInventory.get(item.getId());
 			if(currentInventory <= 0) {
 				String message = "No inventory for " + item.getId();
+				System.out.println(message);
 				throw new RuntimeException(message);
 			}
-
-			tempInventory.put(item.getId(), currentInventory--);
+			currentInventory = currentInventory - 1;
+			tempInventory.put(item.getId(), currentInventory);
 		}
 		inventory = tempInventory;
 	}
 
-	// Class constructor to initialize logging 
 	public void Application() {
 
 	}
 
+	@CrossOrigin(origins = "http://localhost:5000")
 	@PostMapping("/checkout")
-	public void CheckoutCart(@RequestHeader(name = "X-Session-ID", required = true) String sessionId,
-							 @RequestHeader(name = "X-Transaction-ID", required = true) String transactionId,
-							 @RequestBody Order order) {
-
-		// perform checkout
-		
-		// Because MDC is thread dependent, it won't be added
-		// to exceptions that aren't caught
-		//
-		// So we will wrap in try/catch
+	public void CheckoutCart(@RequestBody Order order) {
 		Sentry.configureScope(scope -> {
-			scope.setTag("sessionId", sessionId);
-			scope.setTag("transactionId", transactionId);
-
 			scope.setExtra("cart", order.toJson());
 		});
 
-		MDC.put("sessionId", sessionId);
-		MDC.put("transactionId", transactionId);
```

Comment:
transactionId deprecated as Performance Tracing now takes care of `trace` properties.

`sessionId` deprecated in our tracing demos as well, as we come out with Release Health tracking sessions.

So put two other tags in place, so MDC still gets used here.